src/app/views/overview/overview.component.html

```diff
@@ -12,7 +12,7 @@ <h2>Overview</h2>
         </small>
         <br>
         <small class="text-muted">
-            Older <a href="https://v016.factoriocheatsheet.com/" target="_blank" rel="noopener">
+            The older <a href="https://v016.factoriocheatsheet.com/" target="_blank" rel="noopener">
```

Comment:
Don't think "the" is necessary here or the next line, only wastes space

Reply:
Okay. I thought "the" was neater, but you're in charge here. :+1: